src/rshim_usb.c

```diff
@@ -75,23 +77,34 @@ static void rshim_usb_delete(rshim_backend_t *bd)
   free(dev);
 }
 
+static int chan_to_wValue(int chan, uint16_t ver_id)
+{
+  if (ver_id == RSHIM_BLUEFIELD_3)
+    return chan << 7;
+
+  return chan;
```

Comment:
We might need a table to update both wValue & wIndex for BF3 (by setting the the wValue and adding an offset to wIndex ).

Reply:
done